### PR TITLE
Add separate rate limiting config for webhooks + environment variable support

### PR DESCRIPTION
**Pull request by Rory-Powell**
## Description
- Add separate `/api/webhooks` location + rate limiting config
- Change default rate limit status from 503 to 429
- Add support for environment variables in proxy-service image

### CODE REVIEW

**Review by mike12345567 — APPROVED** [review]
LGTM!

File: hosting/nginx.prod.conf.hbs [hunk 1]
```diff
@@ -126,6 +130,25 @@ http {
       proxy_pass      http://$apps:4002;
     }
 
+    location /api/webhooks/ {
```
Comment (Rory-Powell) [inline hosting/nginx.prod.conf.hbs:133]: Open to suggestions on how to reduce duplication between this location and the above `location /api/`. Nesting doesn't seem to work.

Comment (shogunpurple) [reply hosting/nginx.prod.conf.hbs:133]: You can nest locations in NGINX, but I think it's honestly just cleaner here to have the other route.

**Review by shogunpurple — APPROVED** [review]
LGTM! Just make sure we give this a good test with a locally built image 👍

File: hosting/nginx.prod.conf.hbs (hunk 1 above)
Comment (mike12345567) [reply hosting/nginx.prod.conf.hbs:133]: I would agree, duplication in this case is probably easier to understand from a readability point of view.

**Pull request merged**